src/shell/Shell.cpp

```diff
@@ -77,8 +77,10 @@ ValueRef* builtinPrint(ExecutionStateRef* state, ValueRef* thisValue, size_t arg
     if (argc >= 1) {
         if (argv[0]->isSymbol()) {
             puts(argv[0]->asSymbol()->symbolDescriptiveString()->toStdUTF8String().data());
+            state->context()->printDebugger(argv[0]->asSymbol()->symbolDescriptiveString()->toStdUTF8String().data());
         } else {
             puts(argv[0]->toString(state)->toStdUTF8String().data());
+            state->context()->printDebugger(argv[0]->toString(state)->toStdUTF8String().data());
```

Comment:
Since we send strings here, the API should accept Escargot string, rather than char*.